apps/docs/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@coreproject-moe/icons-docs",
+    "name": "@coreproject-moe/docs",
```

Comment:
Never touch package.json

Reply:
Oke? why. it is not only for icons right?

Reply:
No, dont touch anyhting